src/main/java/sirius/pasta/tagliatelle/emitter/ConstantEmitter.java

```diff
@@ -62,6 +62,33 @@ public ConstantEmitter append(String stringToAppend) {
         return this;
     }
 
+    public ConstantEmitter stripLeading() {
+        if (Strings.isEmpty(value)) {
+            return this;
+        }
+
+        value = value.stripLeading();
+        return this;
+    }
+
+    public ConstantEmitter stripTrailing() {
+        if (Strings.isEmpty(value)) {
+            return this;
+        }
+
+        value = value.stripTrailing();
+        return this;
+    }
+
+    public ConstantEmitter strip() {
+        if (Strings.isEmpty(value)) {
+            return this;
+        }
+
+        value = value.strip();
+        return this;
+    }
```

Comment:
unused, I think?

Reply:
Yes, it is just for the sake of completeness 😅

Reply:
removed